lib/airbrake.rb

```diff
@@ -19,3 +19,28 @@
 require 'airbrake/resque/failure' if defined?(Resque)
 require 'airbrake/sidekiq/error_handler' if defined?(Sidekiq)
 require 'airbrake/delayed_job/plugin' if defined?(Delayed)
+
+##
+# This module defines the Airbrake API and serves as a namespace for other
```

Comment:
Could you please mention that it extends the original class from `airbrake-ruby`? That is, we reopen its definition here.